frontend/benefit/handler/browser-tests/pages/archive.testcafe.ts

```diff
@@ -0,0 +1,42 @@
+import requestLogger, {
+  filterLoggedRequests,
+} from '@frontend/shared/browser-tests/utils/request-logger';
+import { clearDataToPrintOnFailure } from '@frontend/shared/browser-tests/utils/testcafe.utils';
+import ApplicationList from '../page-model/ApplicationList';
+import { RequestMock } from 'testcafe';
+
+import MainIngress from '../page-model/MainIngress';
+import { getFrontendUrl } from '../utils/url.utils';
+
+const getBackendDomain = (): string =>
+  process.env.NEXT_PUBLIC_BACKEND_URL || 'https://localhost:8000';
+
+const url = getFrontendUrl('/archive');
+import jsonArchivedList from '../fixtures/list-archived.json';
```

Comment:
This import fails in the build, should it be `import jsonArchivedList from '../json/list-archived.json'`?